docs/monthly-meeting/2022-10.md

```diff
@@ -0,0 +1,91 @@
+   * next steps: Cam to open a PR in 3.12, also a Discuss thread. Petr to open an issue in Sphinx.
+
+* Docs to support Sphinx 5.x. @CAM-Gerlach wants to add a CI workflow during the sprint week for the oldest-supported version of Sphinx.
+    * Adam will work on sphinx support on python-docs-theme
+
+* (Ezio) Should we enable the default role and alias it to `:literal:` (`` `...` ``) for docs.python.org?
+  * Less typing required, and no more errors caused by MarkDown-like `` `...` `` in rst documents
+  * It's not used for anything, so we could use it instead of (or as an alias to) ` ``...`` `
+  * Unless we mass edit the docs (to switch to single backticks), it might lead to inconsistencies
+  * We could also alias it to `:any:`?
+    * Any projects using that? - [yes](https://github.com/search?l=Python&q=filename%3Aconf.py+default_role+%3D&type=Code) -- `ansible/ansible-lint`, `scipy/scipy` (use a custom "autolink" role), `micropython/micropython`, `python-jsonschema/jsonschema`, etc
+  * `:literal:` is probably best, docs enthusiasts can help with adding proper roles
+  * If we don't want to enable the default role, sphinx-lint already emits an error for single backticks -- making the error message there more friendly might improve usability for new contributors a great deal. (The messsage is currently just "Default role used".)
+
+* [Docs preview PR](https://github.com/python/cpython/pull/92852) is still blocked on Ee allowing access to the Netlify account, which could help Adam's concerns above
+  * **Mariatta** to ping ee again
+
+* Joannah might want to talk about SEO for the docs
+* https://github.com/python/pythondotorg/issues/1691
+    * docs.p.o SEO is not great
+    * discussed in PSF channel
+    * what are tools to improve? How to improve? 
+    * Do we have the skills for improving SEO, or do we need external expert? It may be possible for The PSF to help fund it, we can request it. 
+    * Existing issue: https://github.com/python/pythondotorg/issues/1691
+    * Adam: From Sphinx perspective: there is no SEO there. no expertise from Sphinx.
+    * Petr: we may need a paid expert to handle this
+    * Hugo: add more metadata. There are best practises.
+    * CAM: Google has some tools that will tell you what's missing. Some of the improvements could be done in the theme.
+    * Mariatta: We have a lot of inbound links
+    * Pradyun: there are extensions (e.g. https://pypi.org/project/sphinxext-opengraph/ / https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#html-metadata). re.
+    * e.g. Google Lighthouse audits https://web.dev/measure/?url=https%3A%2F%2Fdocs.python.org%2F3%2Flibrary%2Fcsv.html%3Fhighlight%3Dcsv%23module-csv
+    * CAM: It seems like we should be able to do a lot of this.
+    * Let's do the low-hanging fruit first
+    * First, we need to come up with a list of issues to improve SEO
+    * Then we can see if we need to get an expert paid if need be to advise
```

Comment:
```suggestion
    * Then we can see if we need to get an expert paid if need be to advise
    * @hugovk will open issues for the Lighthouse warnings and any other major fixable ones
```

@hugovk you said you'd try to tackle this, I believe

Reply:
@Mariatta opened a tracking issue here: https://github.com/python/docs-community/issues/63

And a project here: https://github.com/orgs/python/projects/33/views/1

I'm looking into it and will post in that issue.